ansible/vars/db.yml

```diff
@@ -0,0 +1,24 @@
+mysql_database: "v1"
+mysql_root_password: "root"
+mysql_user_password: "root"
+
+mysql_databases:
+  - name: "{{ mysql_database }}"
+    encoding: latin1
```

Comment:
latin1 is the default for mysql.  it's also a single-byte encoding, whereas utf-8 takes 3 bytes.  that said, if you guys are using utf-8, then n/p.  it's an easy fix.